### PR TITLE
[Reddit CAPI Integration] Adding a small fix for hashing and device_type logic

### PR DESCRIPTION
**Pull request by ManikMM**
_A summary of your pull request, including the what change you're making and why._

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Review by joe-ayoub-segment — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
hi @ManikMM PR deployed

**Review by peanutbutterduo — COMMENTED** [review]
@ManikMM For posterity, backfilling the missing review.

File: packages/destination-actions/src/destinations/reddit-conversions-api/utils.ts [hunk 1]
```diff
@@ -114,7 +114,7 @@ function getMetadata(
 function getAdId(device_type?: string, advertising_id?: string): { [key: string]: string | undefined } | undefined {
   if (!device_type) return undefined
   if (!advertising_id) return undefined
-  return device_type === 'Apple' ? { idfa: hash(advertising_id) } : { aaid: hash(advertising_id) }
+  return device_type === 'ios' ? { idfa: hash(advertising_id) } : { aaid: hash(advertising_id) }
```
[the remark this answers is not in the document]
Comment (ManikMM) [reply packages/destination-actions/src/destinations/reddit-conversions-api/utils.ts:117]: @joe-ayoub-segment Would you be able to answer the question above? Is it always going to be: `ios`?
That's what we noticed it was always coming through as but just want to double check now...

[the remark this answers is not in the document]
Comment (joe-ayoub-segment) [reply packages/destination-actions/src/destinations/reddit-conversions-api/utils.ts:117]: hi @ManikMM it will always be lower case.

**Pull request merged**